File: sdkwork-app-sdk-go/types/favorite_type_count_vo.go
package types

// 各类型收藏数量VO
type FavoriteTypeCountVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	Type string `json:"type"`
	Count int `json:"count"`
	TypeName string `json:"typeName"`
}

File: sdkwork-app-sdk-csharp/Models/FavoriteTypeCountVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class FavoriteTypeCountVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public string? Type { get; set; }
        public int? Count { get; set; }
        public string? TypeName { get; set; }
    }
}
